### PR TITLE
Removes horrible sins against nature from cult code

### LINKED ISSUE #3499: No Click Delay when attacking cult constructs
You can spam click constructs to attack them incredibly fast with melee weapons. This results in quick death for any construct other than the behemoth/juggernaut.

### PR DESCRIPTION
**Pull request by Crazylemon64**
Fixes #3499

### CODE REVIEW

File: code/modules/mob/living/simple_animal/constructs.dm [hunk 1]
```diff
@@ -245,8 +210,9 @@
 	speed = 5
 	environment_smash = 2
 	attack_sound = 'sound/weapons/punch4.ogg'
+	force_threshold = 11
 	var/energy = 0
-	var/max_energy = 1000
+	var/max_energy = 100
```
Comment (Crazylemon64) [inline code/modules/mob/living/simple_animal/constructs.dm]: WHAT

File: code/modules/mob/living/simple_animal/constructs.dm [hunk 2]
```diff
@@ -76,22 +76,6 @@
 				O.show_message("<span class='attack'>\The <EM>[M]</EM> [M.attacktext] \the <EM>[src]</EM>!</span>", 1)
 			add_logs(M, src, "attacked")
 			var/damage = rand(M.melee_damage_lower, M.melee_damage_upper)
-			adjustBruteLoss(damage)
```
Comment (Crazylemon64) [inline code/modules/mob/living/simple_animal/constructs.dm]: dagnabbit I ruin everything I touch

**Comment by Plotron** [comment]
Sins against nature?! Naturalistic fallacy!

**Comment by Fox-McCloud** [comment]
Wow. That snowflake.

**Pull request merged**